configuration/configuration.go

```diff
@@ -123,6 +124,22 @@ type Configuration struct {
 	} `yaml:"redis,omitempty"`
 }
 
+// Handlers is for handling error configuration, when some kinds of error happens.
```

Comment:
These are really "hooks" for logging. Yes, they handle errors, but they can do a lot more.